x-pack/plugin/async-search/src/main/java/org/elasticsearch/xpack/search/AsyncSearchTask.java

```diff
@@ -128,7 +129,7 @@ Listener getSearchProgressActionListener() {
      */
     @Override
     public void setExpirationTime(long expirationTimeMillis) {
```

Comment:
Maybe change the name to reflect the new behavior (extendExpirationTimeIfNeeded) ?

Reply:
++, I renamed this in https://github.com/elastic/elasticsearch/pull/67877/commits/369cdc1748079ea0cd33c7301212d8ae5009d614.